docs/migrate-from-legacy-sdks-to-app-bridge.md

```diff
@@ -0,0 +1,248 @@
+# Migrate your app from legacy SDKs to Shopify App Bridge
+
+Shopify has two deprecated client-side app libraries, the [Embedded App SDK (EASDK)](https://shopify.dev/apps/tools/embedded-app-sdk) and the [POS App SDK](https://shopify.dev/apps/tools/pos-app-sdk/), that serve a similar function to Shopify App Bridge.
+
+## Using Shopify App Bridge directly
+
+If you choose to upgrade your app using the Shopify App Bridge actions directly, then you get the following benefits:
+
+- modern front-end development workflows, including modular JavaScript
+- strict versioning using SemVer, for even more stability and predictability
+
+In the future, most new features will be added to [Shopify App Bridge](https://shopify.dev/apps/tools/app-bridge) only, and will not be made available to the EASDK. **We recommend that you upgrade to using Shopify App Bridge if you’re building a new app or doing significant upgrades to your existing app.**
+
+## Migrating your EASDK app
+
+The best place to start is the [Embedded App SDK (EASDK) documentation](https://shopify.dev/apps/tools/embedded-app-sdk/methods), which has been updated with corresponding App Bridge methods. Here are a few additional things to keep in mind when migrating an existing app from EASDK to Shopify App Bridge.
```

Comment:
> which has been updated with corresponding App Bridge methods.

I don't think this is true, I'm confused what this paragraph is referring to 🤔

Reply:
Literally just took it from the removed docs in the dev docs repo 🤷 

Could it be referring to the legacy bridge?

Reply:
Never mind is see the notes now, each one links out to the App Bridge methods 
![image](https://user-images.githubusercontent.com/29458473/129221884-3457015b-e6f9-4f7d-be41-ea995752f648.png)